meta-mel/conf/distro/mel.conf

```diff
@@ -78,9 +78,6 @@ MELDIR ?= "${COREBASE}/.."
 # Use a local PR server by default
 PRSERV_HOST ?= "localhost:0"
 
-# Default to ipk packaging
-PACKAGE_CLASSES ?= "package_ipk"
```

Comment:
What if the default we're relying on moves? Is this something we have picked up consciously in the past? If so, then we should keep this otherwise we're ok.

Reply:
Dropped this commit, in this case we want to be explicit, you're right